Lib/test/test_memoryio.py

```diff
@@ -517,6 +517,23 @@ def test_relative_seek(self):
         memio.seek(1, 1)
         self.assertEqual(memio.read(), buf[1:])
 
+    def test_peek(self):
+        buf = self.buftype("1234567890")
+        memio = self.ioclass(buf)
+
+        self.assertEqual(memio.peek(1), buf[:1])
+        self.assertEqual(memio.peek(1), buf[:1])
```

Comment:
Can you please test the position, test tell()?

Reply:
Sure, done.